docs/scos/dev/setup/quickstart-guides-install-spryker/quickstart-guide-install-spryker-on-macos-and-linux.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Quickstart guide: Install Spryker on MacOS and Linux"
+description: Get started with Spryker on MacOS and Linux
+last_updated: Sep 8, 2022
```

Comment:
If you add something new, update the last_updated tag

Reply:
```suggestion
last_updated: Feb 3, 2022
```